test/test.js

```diff
@@ -27,6 +27,7 @@
 
   /** Used as a reference to the global object. */
   var root = (typeof global == 'object' && global) || this;
+  debugger;
```

Comment:
Whoops! Should have noticed that.

At one point, I had a bug where `baseFlatten` was defaulting `depth` to `0` instead of `1` (because `toInteger(undefined)` behaved differently than I assumed), and the test suite wouldn't even load.